cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTask.java

```diff
@@ -226,7 +272,20 @@ private void ensureValidTransfer(State targetState) {
 
   @Override
   public String toString() {
-    return String.format("{EXE_ID: %d, %s, %s, %s}", _executionId, _type, _proposal, _state);
+    switch (_type) {
+      case INTER_BROKER_REPLICA_ACTION:
+        return String.format("{EXE_ID: %d, %s, %s, %s -> %s, %s}", _executionId, _type, _proposal.topicPartition(),
+                             _proposal.oldReplicas(), _proposal.newReplicas(), _state);
+      case INTRA_BROKER_REPLICA_ACTION:
+        return String.format("{EXE_ID: %d, %s, %s, %s -> %s, %s}", _executionId, _type, _proposal.topicPartition(),
+                             _proposal.oldReplicas().stream().filter(r -> r.brokerId() == _brokerId).collect(Collectors.toList()),
+                             Collections.singletonList(_proposal.replicasToMoveByBroker().get(_brokerId)), _state);
+      case LEADER_ACTION:
+        return String.format("{EXE_ID: %d, %s, %s, %s -> %s, %s}", _executionId, _type, _proposal.topicPartition(),
+                             _proposal.oldLeader(), _proposal.newReplicas().get(0), _state);
+      default:
+        throw new IllegalStateException("Unknown task type " + _type);
+    }
```

Comment:
Can we keep the old response format? As mentioned on other parts, I am afraid this change will cause a lot of memory pressure, and we can sacrifice this missing logdir information.